src/value.cc

```diff
@@ -652,6 +671,8 @@ bool V8ToGIArgument(GIBaseInfo *gi_info, GIArgument *arg, Local<Value> value) {
     }
     case GI_INFO_TYPE_INTERFACE:
         arg->v_pointer = GObjectFromWrapper(value);
+        if (transfer == GI_TRANSFER_EVERYTHING)
+            g_object_ref(arg->v_pointer);
```

Comment:
And this one isn't going to leak anything? Is this case the one tested by the test down here?

Reply:
> And this one isn't going to leak anything?

I suppose no. The new reference should be taken by the callee, by the definition of `TRANSFER_EVERYTHING`. I don't see a failure case in this function.

> Is this case the one tested by the test down here?

Similar, but of different type. The added test is for `GBoxed` [1], while this one is for `GObject`. That's why I said I want to test this.

[1] specifically `GstMiniObject`, but to node-gtk it's a boxed object.

Reply:
> The new reference should be taken by the callee

But maybe here `TRANSFER_EVERYTHING` means to take ownership of the passed ref, no? Either way, I'd first try to find a failing case before adding a reference.